geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/GMSMemberData.java

```diff
@@ -399,11 +399,11 @@ public int hashCode() {
   public String toString() {
     StringBuilder sb = new StringBuilder(100);
 
-    sb.append("MemberData[");
+    sb.append("GMSMember[");
```

Comment:
Why did the name change to `"GMSMember"` here? It was `"MemberData"` which kinda made sense because `GMSMemberData` isa `MemberData`. I could see leaving it alone or maybe making it `GMSMemberData`.

Reply:
I think that's an artifact of my doing this work on an old SHA and then rebasing onto develop.  I'll revert that change